packages/core/src/_typography.scss

```diff
@@ -281,6 +285,43 @@ Styleguide preformatted
   @extend %code-block;
 }
 
+%keyboard {
+  $kbd-key-size: $pt-button-height-small;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: $pt-border-radius;
+  box-shadow: $pt-elevation-shadow-1;
+  background: $white;
+  min-width: $kbd-key-size;
+  height: $kbd-key-size;
+  padding: ($pt-grid-size * 0.3) ($pt-grid-size / 2);
```

Comment:
use `3px 6px` (`$pt-border-radius ($pt-border-radius * 2)`) and remove modifier-key overrides below